carwings.net/Carwings.cs

```diff
@@ -32,80 +33,140 @@ public Carwings()
             };
         }
 
-        public async Task<Vehicle[]> Login(string userid, string password)
+        /// <summary>
+        /// Log in to the specified Nissan account
+        /// </summary>
+        /// <param name="userid">Email address</param>
+        /// <param name="password">password</param>
+        /// <param name="country">Country or Nissan region name, different from Nissan's V1 API.  Valid values are USA: "US"</param>
+        /// <param name="cancellationToken">Support for cancellation.</param>
+        /// <returns>An array of Vehicles for this user.  If the login credentials were wrong, it returns a null Vehicle[].</returns>
+        [Obsolete("Provide a cancellation token")]
```

Comment:
i'm not sure i'd worry about back compat here

Reply:
I was hoping to not totally trash your existing usages of this library.  But I'll get rid of the method instead.